src/content/en/showcase/2016/service-worker-perf.md

```diff
@@ -0,0 +1,679 @@
+
+To include this information in all hits (e.g. all pageviews, events, etc.) we [set](/analytics/devguides/collection/analyticsjs/accessing-trackers#updating_tracker_data) the custom dimension value on the [tracker](/analytics/devguides/collection/analyticsjs/creating-trackers) object itself, prior to sending any data to Google Analytics.
+
+<pre class="prettyprint">
+ga('set', 'dimension1', getServiceWorkerStatus());
+</pre>
+
+Once set, this value gets sent with all subsequent hits for the current pageload. If the user load the page again later, a new value will likely be returned from the `getServiceWorkerStatus` function, and that value will be set on the tracker object.
+
+A quick note on code clarity and readability: since other people looking at this code may not know what `dimension1` refers to, it's always best to create a variable that maps meaningful dimension names to the values analytics.js will use.
+
+<pre class="prettyprint">
+<strong>// Creates a map between custom dimension names and their index.
+// This is particularly useful if you define lots of custom dimensions.
+var customDimensions = {
+  SERVICE_WORKER_STATUS: 'dimension1'
+};</strong>
+
+// Creates the tracker object.
+ga('create', 'UA-XXXXX-Y', 'auto');
+
+<strong>// Sets the service worker status on the tracker,
+// so its value is included in all future hits.
+ga('set', customDimensions.SERVICE_WORKER_STATUS, getServiceWorkerStatus());</strong>
+
+// Postpones sending any hits until after the page has fully loaded.
+// This prevents analytics requests from delaying the loading of the page.
+window.addEventListener('load', function() {
+  // Sends a pageview for the initial pageload.
+  ga('send', 'pageview');
+
+  // Sends an event with the time to first paint data.
+  sendTimeToFirstPaint();
+});
+</pre>
+
+As I mentioned, sending the *Service Worker Status* dimension with every hit allows us to use it when reporting on any metric. For example, here's what service worker usage looked like for all pageviews during the week of Google I/O:
+
+<iframe width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/a/google.com/spreadsheets/d/18uCZKpkXii3eD2Z9pzqNdIYa8sjCI-zOnHjr5KkZmaE/pubchart?oid=537990599&amp;format=interactive"></iframe>
+
+As you can see almost 85% of all pageviews for IOWA were from browsers that [support service worker](https://jakearchibald.github.io/isserviceworkerready/).
+
+## The results: answering our questions
+
+Once we updated our tracking implementation to track the data needed to answer our original questions, we could start reporting on that data to see the results. *(Note: all Google Analytics data shown here represents actual web traffic to the IOWA site from May 16-22, 2016).*
```

Comment:
Suggest shortening 'Once we updated our tracking implementation to track the data needed to answer our original questions' to 'Once we started tracking to answer our questions.'